plugins/chat/assets/stylesheets/common/chat-messages-scroller.scss

```diff
@@ -11,4 +11,5 @@
   min-height: 1px;
   box-sizing: border-box;
   transition: padding-top 0.2s ease-in-out;
+  height: 1px;
```

Comment:
a small hack to prevent loading state of channel to take too much space and hide the navbar